client/sites-dashboard/components/sites-table-row.tsx

```diff
@@ -79,6 +88,16 @@ const VisitDashboardItem = ( site: Site ) => {
 
 export default function SitesTableRow( { site }: SiteTableRowProps ) {
 	const { __ } = useI18n();
+
+	const isComingSoon =
+		site.is_coming_soon || ( site.is_private && site.launch_status === 'unlaunched' );
```

Comment:
Why are we defining this condition here? Does it make sense to lift this derived state up?

Reply:
> Why are we defining this condition here? Does it make sense to lift this derived state up?

@zaguiini Where would you suggest putting it?

Reply:
Ideally, we'd have a data processing function so we could store the derived state and use it throughout the application. That way, other components would benefit from that new state and we could avoid repeating ourselves.

But that's basically impossible right now, as other components are using the data through the `getSites` selector. We could introduce new selectors or plain, utility functions, that would abstract the conditionals.